app/serializers/item_serializer.rb

```diff
@@ -1,3 +1,7 @@
 class ItemSerializer < ActiveModel::Serializer
   attributes :id, :name, :description, :unit_price, :merchant_id
+
+  def unit_price
+    object.unit_price.to_f / 100
+  end
 end
```

Comment:
@cmacaulay I *like* that it's just math! No need for string manipulation when we have math! Double check to make sure that it consistently returns two decimals, but otherwise think this is great.